legal-api/requirements.txt

```diff
@@ -59,4 +59,6 @@ minio==7.0.2
 PyPDF2==1.26.0
 reportlab==3.6.12
 html-sanitizer==1.9.3
-git+https://github.com/bcgov/business-schemas.git@2.18.8#egg=registry_schemas
+# TODO
+# git+https://github.com/bcgov/business-schemas.git@2.18.8#egg=registry_schemas
+git+https://github.com/Jxio/business-schemas.git#egg=registry_schemas
```

Comment:
I'm leaving this comment here so you remember to remove this before the PR is merged. After you have deleted this, you can resolve this comment.